release-notes/1.0/1.0.19189.1-Preview.md

```diff
@@ -0,0 +1,159 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 1.0.19189.1 Preview released 8 July 2019
+
+This update does not add any major features over the previous preview release but includes the below fixes and changes.
+
+### Target Platform Support
+
+* .NET Framework 4.6+
+* .NET Core 2.1+ (Windows x86, Windows x64, Linux, macOS)
+* .NET Standard 2.0+ (Windows x86, Windows x64, Linux, macOS)
+
+### Dependencies
+
+#### .NET Framework
+
+* System.Data.Common 4.3.0
+* Microsoft.Data.SqlClient.SNI
+
+#### .NET Core
+
+* Microsoft.Win32.Registry 4.5.0
+* runtime.native.System.Data.SqlClient.sni 4.4.0
+* System.Security.Principal.Windows 4.5.0
+* System.Text.Encoding.CodePages 4.5.0
+* System.Configuration.ConfigurationManager 4.5.0
+
+#### .NET Standard
+
+* Microsoft.Win32.Registry 4.5.0
+* runtime.native.System.Data.SqlClient.sni 4.4.0
+* System.Buffers 4.4.0
+* System.Diagnostics.DiagnosticSource 4.5.0
+* System.Memory 4.5.1
+* System.Security.Principal.Windows 4.5.0
+* System.Text.Encoding.CodePages 4.5.0
+* System.Configuration.ConfigurationManager 4.5.0
+
+### Fixed
+
+* Microsoft.SqlServer.Server conflicts between System.Data.SqlClient and Microsoft.Data.SqlClient [SqlClient Issue #4](https://github.com/dotnet/SqlClient/issues/4)
+  SqlDataRecord, SqlMetaData, and UDT related classes were moved into Microsoft.Data.SqlClient.Server to avoid namespace conflicts. While UDT related classes are there, UDTs do not currently work in Microsoft.Data.SqlClient.
+* Committing certain data results in an infinite loop inside System.Data.SqlClient.TdsParserStateObject.cs. Ported from [CoreFx PR 34709](https://github.com/dotnet/corefx/pull/34709)
+* When targeting **.NET Framework** and platform **AnyCPU** or **x86**, you may see a build warning about processor architecture mismatch: *Warning	There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "Microsoft.Data.SqlClient*...
```

Comment:
Maybe this line needs a bit of rewording, with the tab spacing after warning and "Microsoft.Data.SqlClient*

Reply:
I see this in the previous two release notes too. Don't we have solved this by splitting SNI native code from our driver's code?